pkg/unpack/oci.go

```diff
@@ -74,17 +74,10 @@ func WithRsyncFlagsOCI(flags ...string) OCIOpt {
 }
 
 func NewOCIUnpacker(s *sys.System, imageRef string, opts ...OCIOpt) *OCI {
-	unpacker := &OCI{
-		s:           s,
-		verify:      true,
-		platformRef: s.Platform().String(),
```

Comment:
I think this line was here to set the value to the host platform by default, but it doesn't work. And anyway having it defined in the `cli/cmd` side is better IMO and we can see the default value with an `elemental3ctl unpack-image --help`.

Reply:
In fact it is still needed when not used with `unpack-image`, but we can still set a default value for it.